client/state/reader/tags/items/actions.js

```diff
@@ -20,7 +19,9 @@ import 'calypso/state/reader/init';
  * @returns {string}      Tag slug
  */
 export const slugify = ( tag ) =>
-	encodeURIComponent( trim( tag ).toLowerCase().replace( /\s+/g, '-' ).replace( /-{2,}/g, '-' ) );
+	typeof tag === 'string'
+		? encodeURIComponent( tag.trim().toLowerCase().replace( /\s+/g, '-' ).replace( /-{2,}/g, '-' ) )
+		: '';
```

Comment:
Seems to me that the `tag` param is always safely a string. The `requestTags` action does its own null check, and the `requestFollow`/`requestUnfollow` actions don't make sense without a tag.

Reply:
It may be a `null` though:

https://github.com/Automattic/wp-calypso/blob/336b63bee826814cdc6f4f51d7b3b9b6f7a2334d/client/components/data/query-reader-followed-tags/index.jsx#L15

But in that case, it might be more concise to go with optional chaining and nullish coalescing.

Reply:
Yes, but `requestTags` is already covered, it has its own `tag ? slugify( tag ) : null` condition.